Cargo.toml

```diff
@@ -5,7 +5,7 @@ members = ["crates/*"]
 # to inherit from this rather than needing to duplicate its content themselves.  Until we get there,
 # keep this in sync with each subpackage `Cargo.toml`'s `package` key.
 [workspace.package]
-version = "0.24.0"
+version = "0.24.1"
```

Comment:
For publicity as well: starting from Terra 0.26, we'll be able to raise the MSRV sufficiently high that we can inherit these properties into the individual crates from the workspace's `Cargo.toml`.  In the meantime, we have to do it manually.

Reply:
Good. I have been updating notes on this procedure, I'll add these files to them. (I know it may go out of date, but I still think it's useful.)

I noticed `Cargo.lock` also carries the version number. Of course it should be updated. That will go in notes as well.